tests/brevitas/core/test_bit_width.py

```diff
@@ -123,3 +124,15 @@ def test_override_pretrained_value(self, bit_width_parameter, override_pretraine
             assert value == override_value
         else:
             assert value == state_dict_value
+
+    def test_load_from_stateful_const(self, bit_width_parameter, override_pretrained):
```

Comment:
once we define `bit_width_init_two`, we can have as fixtures `bit_width_stateful_const`, `bit_width_parameter`, `bit_width_init`, `override_pretrained`.